rpi_backlight.py

```diff
@@ -45,38 +46,53 @@ def _set_value(name: str, value: Any) -> None:
             _perm_denied()
 
 
+def set_brightness_value(value: int) -> None:
+    if MODE == "MODE_TINKERBOARD":
+        _set_value("tinker_mcu_bl", value)
+    elif MODE == "MODE_RPI":
+        _set_value("bl_power", value)
+
 def get_actual_brightness() -> int:
     """Return the actual display brightness."""
-    return int(_get_value("actual_brightness"))
-
+    if MODE == "MODE_TINKERBOARD":
+        return int(_get_value("tinker_mcu_bl"))
+    elif MODE == "MODE_RPI":
+        return int(_get_value("actual_brightness"))
 
 def get_max_brightness() -> int:
     """Return the maximum display brightness."""
-    return int(_get_value("max_brightness"))
-
+    if MODE == "MODE_TINKERBOARD":
+        return 255
+    elif MODE == "MODE_RPI":
+        return int(_get_value("max_brightness"))
+    
 
 def get_power() -> bool:
-    """Return wether the display is powered on or not."""
+    """Return whether the display is powered on or not."""
     # 0 is on, 1 is off
-    return not int(_get_value("bl_power"))
+    if MODE == "MODE_TINKERBOARD":
+        if get_actual_brightness():
+            return True
+        else:
+            return False
+    elif MODE == "MODE_RPI":
+        return not int(_get_value("bl_power"))
 
 
-def set_brightness(value: int, smooth: bool = False, duration: float = 1) -> None:
+def set_brightness(value: int, smooth: bool = True, duration: float = 1) -> None:
```

Comment:
I'd prefer to not change this for now, at least not in this PR - which should be only about adding support for the Tinkerboard.

Reply:
fixed